src/python/stubs/OpenImageIO/__init__.pyi

```diff
@@ -0,0 +1,1617 @@
+#
+# This file is auto-generated. DO NOT MODIFY!  Run `make pystubs` to regenerate
+#
+
```

Comment:
I'm ok with checking this into the repo, but do we have an automated way to detect that something has changed about the python bindings in a way that necessitates a re-generation? Should we add a note to the RELEASING.md guide reminding us to regenerate on every release as part of the release staging process?

Reply:
If a developer changes anything in the code that results in the type stubs changes, the `cp311-manylinux_x86_64` 
 and `cp311-manylinux_aarch64` while build jobs in CI will fail, and will print the instructions on how to update the stubs.  I also updated the INSTALL.md with the instructions on how to update the stubs.

Reply:
ok, good enough